src/main/java/mate/academy/intro/repository/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.intro.repository;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.intro.dto.BookSearchParametersDto;
+import mate.academy.intro.model.Book;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    private final String title = "title";
```

Comment:
follow the naming convention for constants 
same for other strings
```suggestion
    private static final String TITLE_COLUMN = "title";
```